pkg/parser/parser.go

```diff
@@ -1,42 +1,121 @@
 package parser
 
 import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"path/filepath"
+
 	"github.com/instrumenta/conftest/pkg/parser/cue"
 	"github.com/instrumenta/conftest/pkg/parser/ini"
 	"github.com/instrumenta/conftest/pkg/parser/terraform"
 	"github.com/instrumenta/conftest/pkg/parser/toml"
 	"github.com/instrumenta/conftest/pkg/parser/yaml"
 )
 
+// ValidInputs returns string array in order to passing valid input types to viper
+func ValidInputs() []string {
```

Comment:
This looks like it's missing `json`, which may have been an existing issue but hard to track given the diff.

Reply:
I went ahead and added it, the `yaml.go` file takes care of parsing `json` input